engine/apps/alerts/models/alert_group.py

```diff
@@ -648,11 +653,7 @@ def acknowledge_by_user(self, user: User, action_source: typing.Optional[ActionS
             f"log record {log_record.pk} with type '{log_record.get_type_display()}', action source: {action_source}"
         )
 
-        alert_group_action_triggered_signal.send(
-            sender=self.acknowledge_by_user,
-            log_record=log_record.pk,
-            action_source=action_source,
```

Comment:
I'll double-check this, I think only slack uses action_source from the signal but force_sync was used instead (for the delete case).  In all cases it should be able to be taken from the log record instead of it being passed on the signal, I wanted to avoid changing the signature on send_alert_group_signal/having to add another task with a different signature.